data/soapvalidator/Admin/Accounts/DeleteAccount_WithMsg.xml

```diff
@@ -0,0 +1,119 @@
+<t:tests
+    xmlns:t="urn:zimbraTestHarness">
+    <t:property name="test_acct1.server" value="NOT_DEFINED"/>
+    <t:property name="test_account1.name" value="test${TIME}.${COUNTER}@${defaultdomain.name}"/>
+    <t:property name="test_account1.password" value="${defaultpassword.value}"/>
+    <t:property name="compose.subject" value="Subject of the message is testing"/>
+    <t:property name="compose.content" value="Content in the message is contents..."/>
+    <t:property name="server.zimbraAdmin" value="${zimbraServer.name}"/>
+    <t:property name="test_acct.server" value="NOT_DEFINED"/>
+    <t:test_case testcaseid="ping" type="always" >
+        <t:objective>basic system check</t:objective>
+        <t:test id="ping" required="true">
+            <t:request>
+                <PingRequest
+                    xmlns="urn:zimbraAdmin"/>
+                </t:request>
+                <t:response>
+                    <t:select path="//admin:PingResponse"/>
+                </t:response>
+            </t:test>
+        </t:test_case>
+        <t:test_case testcaseid="acctSetup1_send_message" type="always" >
+            <t:objective>create test account</t:objective>
+            <t:test id="admin_login" required="true" >
+                <t:request>
+                    <AuthRequest
+                        xmlns="urn:zimbraAdmin">
+                        <name>${admin.user}</name>
+                        <password>${admin.password}</password>
+                    </AuthRequest>
+                </t:request>
+                <t:response>
+                    <t:select path="//admin:AuthResponse/admin:authToken" set="authToken"/>
+                </t:response>
+            </t:test>
+            <t:test required="true" >
+                <t:request>
+                    <CreateAccountRequest
+                        xmlns="urn:zimbraAdmin">
+                        <name>${test_account1.name}</name>
+                        <password>${test_account1.password}</password>
+                    </CreateAccountRequest>
+                </t:request>
+                <t:response>
+                    <t:select path="//admin:CreateAccountResponse/admin:account" attr="id"  set="test_account1.id"/>
+                    <t:select path='//admin:CreateAccountResponse/admin:account/admin:a[@n="zimbraMailHost"]' set="test_acct.server"/>
```

Comment:
Please remove all unused properties like test_acct1.server, test_acct1.server, server.zimbraAccount. Also, remove setting these properties in CreateAccountResponse

Reply:
done